vscode-lean4/src/rpc.ts

```diff
@@ -39,8 +39,15 @@ export class Rpc {
                 try {
                     this.sendMessage({ seqNum, result: await this.methods[name](...args) })
                 } catch (ex) {
-                    // TODO(WN): somehow `ex.message` is lost here on going through `webview.postMessage`
-                    this.sendMessage({ seqNum, exception: ex === undefined ? 'error' : ex })
+                    if (ex === undefined) {
+                        this.sendMessage({ seqNum, exception: 'error' })
+                        return
+                    }
+                    /* Somehow certain properties (such as `ex.message`) disappear along the way
+                     * through `Webview.postMessage` but make it through in a new object. */
```

Comment:
This is interesting.  I wanted to suggest using `{...ex}`, but look what happens:
```
> {...new Error('my message')}
{}
```

After studying the ECMA specification, I believe this because the `message` attribute is not *enumerable*:
```
> Object.getOwnPropertyDescriptor(new Error('my message'), 'message')
{
  value: 'my message',
  writable: true,
  enumerable: false,
  configurable: true
}
```